CodeGeneration/Sources/generate-swiftsyntaxbuilder/BuildableCollectionNodesFile.swift

```diff
@@ -107,153 +102,80 @@ let buildableCollectionNodesFile = SourceFile {
 private func createArrayInitializer(node: Node) -> InitializerDecl {
   let type = node.type
   let elementType = node.collectionElementType
-  return InitializerDecl(
-    leadingTrivia: [
-      "/// Creates a `\(type.buildableBaseName)` with the provided list of elements.",
-      "/// - Parameters:",
-      "///   - elements: A list of `\(elementType.expressibleAsBaseName)`",
-    ].map { .docLineComment($0) + .newline }.reduce([], +),
-    modifiers: [Token.public],
-    signature: FunctionSignature(
-      input: ParameterClause {
-        FunctionParameter(
-          firstName: .wildcard,
-          secondName: .identifier("elements"),
-          colon: .colon,
-          type: ArrayType(elementType: elementType.expressibleAs)
-        )
-      }
+  var elementsInit: ExprBuildable = IdentifierExpr("elements")
+  if !elementType.isToken {
+    elementsInit = FunctionCallExpr(
+      calledExpression: MemberAccessExpr(base: elementsInit, name: "map"),
+      trailingClosure: ClosureExpr(" { $0.create\(elementType.buildableBaseName)() }")
     )
-  ) {
-    SequenceExpr {
-      MemberAccessExpr(base: "self", name: "elements")
-      AssignmentExpr()
-      if elementType.isToken {
-        "elements"
-      } else {
-        FunctionCallExpr(MemberAccessExpr(base: "elements", name: "map"), trailingClosure: ClosureExpr {
-          FunctionCallExpr(MemberAccessExpr(base: "$0", name: "create\(elementType.buildableBaseName)"))
-        })
-      }
-    }
   }
+  return InitializerDecl(
+    """
+    /// Creates a `\(type.buildableBaseName)` with the provided list of elements.
+    /// - Parameters:
+    ///   - elements: A list of `\(elementType.expressibleAsBaseName)`
+    public init(_ elements: \(ArrayType(elementType: elementType.expressibleAs))) {
+      self.elements = \(elementsInit)
```

Comment:
Annoyingly, this uses implicit opening of existentials, so it fails with older toolchains.

Reply:
(I have a fix)